week-3/extended-project/nowhere/views/about.erb

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>NoWhere | About us</title>
+    <link
+      rel="icon"
+      type="image/png"
+      href="https://i.imgur.com/3t8FYL1.png"
+      sizes="32x32"
+    />
+
+    <!-- Custom-Files -->
+    <link rel="stylesheet" href="css/bootstrap.css" />
+    <!-- Bootstrap-Core-CSS -->
+    <link rel="stylesheet" href="css/style.css" type="text/css" media="all" />
+    <!-- Style-CSS -->
+    <!-- font-awesome-icons -->
+    <link href="css/font-awesome.css" rel="stylesheet" />

```

Comment:
I would recommend when you add links to files, you keep a format to put attributes in a specify order, in the line 16 and 18 you use the "rel" attribute as first, but in the line 21 change a another attribute. If it would have begin with rel, it will be more readable to the first view.